blueprint/cli/src/bin/db.rs

```diff
@@ -113,6 +119,45 @@ async fn cli() {
                     }
                 }
             }
+            Commands::Prepare => {
+                if let Err(e) = ensure_sqlx_cli_installed(&mut ui).await {
+                    ui.error("Error ensuring sqlx-cli is installed!", e);
+                    return;
+                }
+                
+                let cargo = get_cargo_path().expect("Existence of CARGO env var is asserted by calling `ensure_sqlx_cli_installed`");
+
+                let mut sqlx_prepare_command = {
+                    let mut cmd = tokio::process::Command::new(&cargo);
+                    cmd.args(["sqlx", "prepare", "--", "--all-targets", "--all-features"]);
```

Comment:
I think that it needs to have a way to pass a `--check` option and `--workspace` option.

Reply:
I'm not so sure. 

- The `--check` mode simply has `prepare` exit with a 1 exit code in case query metadata needs updating. However, this is covered by running cargo clippy in offline mode: this will fail in case the metadata is not up to date.
- The `--workspace` flag gives users flexibility, but Gerust is set up in such a way that every database query lives in the `db` crate (correct me if I'm wrong @marcoow), as do the migrations, and therefore for consistency we should require the query metadata to be placed inside that crate as well.